packages/clerk-js/src/core/constants.ts

```diff
@@ -1,4 +1,11 @@
-export const PRESERVED_QUERYSTRING_PARAMS = ['after_sign_in_url', 'after_sign_up_url', 'redirect_url'];
+// TODO: Do we still have a use for this or can we simply preserve all params?
+export const PRESERVED_QUERYSTRING_PARAMS = [
```

Comment:
This is the list of search params we preserve when navigating using our internal router. I don't recall any reason that'd block us from simply preserving all params (including user-passed params)